chia/full_node/mempool_manager.py

```diff
@@ -124,7 +124,7 @@ def compute_assert_height(
 
 @dataclass
 class SpendBundleAddInfo:
-    cost: Optional[uint64]
+    cost: uint64
```

Comment:
This is unrelated to the patch, it's a result of altering `add_spendbundle`'s failure path (addressed in the addendum).